adage-front/src/app/components/OffersSearch/Offers/PrebookingButton/PrebookingButton.tsx

```diff
@@ -0,0 +1,41 @@
+import { NotificationType, Notification,
+    NotificationComponent, } from 'app/components/Layout/Notification/Notification';
+import React, { useCallback, useState } from 'react';
+import { preBookStock } from 'repository/pcapi/pcapi';
+import { StockType } from 'utils/types';
+import './PrebookingButton.scss';
+
+const PrebookingButton = ({className, stock, canPrebookOffers}: {className?: string, stock: StockType, canPrebookOffers: boolean}) => {
+    const [disableButton, setDisableButton] = useState(false)
```

Comment:
par convention les boolean commencent par is, has, should, ou un truc du genre
```suggestion
    const [isButtonDisabled, setButtonDisabled] = useState(false)
```